storage/blob.go

```diff
@@ -119,6 +119,7 @@ type BlobProperties struct {
 	CopyProgress          string   `xml:"CopyProgress"`
 	CopyCompletionTime    string   `xml:"CopyCompletionTime"`
 	CopyStatusDescription string   `xml:"CopyStatusDescription"`
+	LeaseStatus           string   `xml:"x-ms-lease-status"`
```

Comment:
Sorry, I misunderstood the question. That's a good catch!

Response headers of GetBlobProperties contains the Lease Status under the header: **x-ms-lease-status**

But, the Xml of ListBlob contains the Lease Status under the tag: **LeaseStatus** as seen at [ListBlobs documentation](https://msdn.microsoft.com/en-us/library/azure/dd135734.aspx). I'm modifying the tag name accordingly.